src/main/java/step5/domain/result/CarMovementRoundResults.java

```diff
@@ -0,0 +1,25 @@
+package step5.domain.result;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+public class CarMovementRoundResults {
+    private final List<CarMovementRoundResult> results = new ArrayList<>();
+
+    public void add(CarMovementRoundResult roundResult) {
+        results.add(roundResult);
+    }
+
+    public void forEach(Consumer<CarMovementRoundResult> action) {
+        results.forEach(action);
+    }
+
+    public Optional<CarMovementRoundResult> lastRoundResult() {
+        if (results.isEmpty()) {
+            return Optional.empty();
+        }
+        return Optional.of(results.get(results.size() - 1));
```

Comment:
Nit: `results`의 자료형을 `LinkedList`로 변경하면 `getLast()`로 마지막 라운드를 쉽게 꺼낼 수 있을 것 같아요.
아니면 `results.get(results.size() - 1)`을 메서드로 추출하여 직관적인 이름을 부여하는 방법도 있겠네요.
```suggestion
        return Optional.of(getLastResult());
```
  
그리고 `CarMovementRoundResults`에서 곧바로 우승자들의 이름을 반환하는 것도 나쁘지 않을 것 같아요.
```java
public List<String> winnerNames() {
    if (results.isEmpty()) {
        return Collections.emptyList();
    }
    return results.getLast().winnerNames();
}
```